src/components/commentList.js

```diff
@@ -0,0 +1,17 @@
+export const countComments = (comments) => comments.length;
+
+const commentList = (comments) => {
+  const commentCount = document.getElementById('commentCount');
+  commentCount.innerHTML = `${countComments(comments)}`;
+
+  const commentList = document.querySelector('.comment-list');
+  commentList.innerHTML = comments.map((comment) => `
+          <div class="comment">
+            <div class="comment-author">${comment.username}</div>
+            <div class="comment-date">${comment.creation_date}</div>
+            <div class="comment-content">${comment.comment}</div>
```

Comment:
- [ ] Please kindly make sure the inputs are  cleared after submitting the comments,  and that the `textarea` is clear when it is active. Please make sure the text in the `textarea` is a `placeholder`. See image  below.

![placeholder](https://user-images.githubusercontent.com/65852621/143441353-a656956b-d3f2-457b-b999-3cb9a7e531f8.png)